sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -366,6 +369,58 @@ def test_model_pipeline_same_as_normalize_true(LinearModel, params):
     assert_allclose(y_pred_normalize, y_pred_standardize)
 
 
+# FIXME: 'normalize' to be removed in 1.2
+@pytest.mark.filterwarnings("ignore:'normalize' was deprecated")
+@pytest.mark.parametrize(
+    "estimator, is_sparse",
+    [(LinearRegression, True),
+     (LinearRegression, False)]
```

Comment:
If this test is only meant to test `LinearRegression` it should be moved to `sklearn/linear_model/tests/test_base.py`. If it's meant to be extended to `Ridge`, `Lasso`... maybe it should be move to a new file, e.g. `sklearn/linear_model/tests/test_linear_model.py`